core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractBoxable.java

```diff
@@ -44,19 +41,14 @@ public void setDiscarded(boolean discarded) {
   }
 
   @Override
-  public String getAlias() {
-    return alias;
-  }
+  public abstract String getAlias();
 
   @Override
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "setAlias" is defined in the "Boxable" interface and can be removed from this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3038)